.github/workflows/ci-riscv64.yml

```diff
@@ -70,14 +71,16 @@ jobs:
     # Install cross-compiler for cross-compiling Linux build.
     # Unfortunately there are no libunwind or compression cross-compile
     # packages so we unpack the native versions and copy their files.
-    # Unfortunately, only kinetic (ubuntu 22.10) has libunwind-riscv64 packages.
-    # I prefer debian-ports here, it is closer to upstream.
     - name: Create Build Environment
       run: |
         sudo apt-get update
         sudo apt-get -y install doxygen vera++ cmake crossbuild-essential-riscv64 git  \
           qemu-user qemu-user-binfmt
-        sudo add-apt-repository 'deb [trusted=yes arch=riscv64] http://deb.debian.org/debian sid main'
+        # We cannot use the following command here, because it will download the
+        # x86_64 binaries due to some known issue of DEB822.
+        # sudo add-apt-repository 'deb [arch=riscv64] http://ports.ubuntu.com/ubuntu-ports noble main'
+        echo 'deb [arch=riscv64] http://ports.ubuntu.com/ubuntu-ports noble main' | sudo tee -a /etc/apt/sources.list
```

Comment:
Please update the commands in api/docs/building.dox which are using add-apt-repository with sid.